test/lsp.py

```diff
@@ -272,27 +273,34 @@ def create_cli_parser() -> argparse.ArgumentParser:
     parser = argparse.ArgumentParser(description="Solidity LSP Test suite")
     parser.set_defaults(fail_fast=False)
     parser.add_argument(
-        "-f, --fail-fast",
+        "-f", "--fail-fast",
         dest="fail_fast",
         action="store_true",
         help="Terminates the running tests on first failure."
     )
+    parser.set_defaults(batch=False)
+    parser.add_argument(
+        "-b", "--batch",
+        dest="batch",
+        action="store_true",
+        help="Prevent interactive queries and just fail instead."
```

Comment:
Honestly, is `--batch` then the right word to go with? Why not `--non-interactive` to avoid interactive use (such as in CI runs)?